shell/platform/darwin/ios/framework/Source/FlutterTextInputPluginTest.mm

```diff
@@ -1426,44 +1427,53 @@ - (void)testUpdateFirstRectForRange {
     @(-6.0), @(3.0), @(9.0), @(1.0)
   ];
 
+  CGRect kInvalidFirstRectRelative =
+      [textInputPlugin.viewController.view convertRect:kInvalidFirstRect toView:inputView];
```

Comment:
Wanna highlight changes to this test. Let me know if it makes sense @LongCatIsLooong